src/main/java/com/epam/ta/reportportal/ws/model/project/config/pattern/CreatePatternTemplateRQ.java

```diff
@@ -0,0 +1,79 @@
+package com.epam.ta.reportportal.ws.model.project.config.pattern;
+
+import com.epam.ta.reportportal.ws.annotations.NotEmpty;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+import static com.epam.ta.reportportal.ws.model.ValidationConstraints.MAX_ANALYSIS_PATTERN_NAME_LENGTH;
+import static com.epam.ta.reportportal.ws.model.ValidationConstraints.MIN_ANALYSIS_PATTERN_NAME_LENGTH;
+
+/**
+ * @author <a href="mailto:ivan_budayeu@epam.com">Ivan Budayeu</a>
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class CreatePatternTemplateRQ {
+
+	@NotNull
+	@NotEmpty
```

Comment:
`@NotEmpty` checks for null, so you can remove `@NotNull`

Reply:
NotEmptyValidator allows the value to be 'null', but only if it's 'not null' validation constraint will check whether the String is empty or not